### PR TITLE
Remove utilities module

### PR DESCRIPTION
**Pull request by eivindjahren**

### CODE REVIEW

File: snapwell/snapecl.py [hunk 1]
```diff
@@ -14,11 +14,75 @@
 
 
 import logging
+from math import inf, isinf, isnan, nan, sqrt
 
-from .snap_utils import Inf, Nan, dist, enterSnapMode, findKeyword, roundAwayFromEven
 from .snapconfig import OwcDefinition
 
 
+def dist(p1, p2):
```
Comment (oyvindeide) [inline snapwell/snapecl.py:22]: `math` should already have a `dist`?

Comment (eivindjahren) [reply snapwell/snapecl.py]: Indeed they do, fixed so that it uses `math.dist` .

File: snapwell/wellpath.py [hunk 2]
```diff
@@ -14,11 +14,37 @@
...
 
-from .snap_utils import Inf, close, finiteFloat
-from .snap_utils import read_next_tokenline as token
-from .snap_utils import tryFloat
+
+def finiteFloat(elt):
+    return not (isinf(elt) or isnan(elt))
+
+
+def _ignorable_(line):
```
Comment (oyvindeide) [inline snapwell/wellpath.py:25]: Dont think this (and what uses it) is actually used, must be from the old config.

Comment (eivindjahren) [reply snapwell/wellpath.py:25]: The parsing is flaky and it is used in some constructed test-cases. It ensures that all lines with only comments are skipped. Line-comments are not implemented.

**Review by oyvindeide — APPROVED** [review]
LGTM!

**Pull request merged**